.github/workflows/build-test.yml

```
@@ -44,7 +44,7 @@ jobs:
     name: Build & Test
     needs:
       - prepare-environment
-    uses: open-space-collective/open-space-toolkit/.github/workflows/build-test.yml@main
+    uses: open-space-collective/open-space-toolkit/.github/workflows/build-test.yml@chore/improve-dev-container
```

```suggestion
    uses: open-space-collective/open-space-toolkit/.github/workflows/build-test.yml@main
```
TODO: remove before release